deployment/ansible/development/roles/docker_compose/templates/docker-compose.override.yml.j2

```diff
@@ -8,6 +8,13 @@ services:
       - ../inasafe:/usr/share/qgis/python/plugins/inasafe
       - ../src:/home/app
       - ./pycharm-helpers/inasafe-headless:/root/.pycharm_helpers
+
+    {% if 'environment' in inasafe_headless_worker %}
+    environment:
```

Comment:
have you tried this?

give new line before environment. it is important.

Reply:
It's working fine for me. What's the problem in yours @lucernae ?

Reply:
You need to put newline before and after jinja switch statement.
If not, then ansible will think it was one line so `environment` will end up missing opening tabs/spaces --> wrong yaml format.

You didn't encounter this issue because your setup ansible is not using `environment` key in your local machine and travis. Hence, this statement is skipped.

Reply:
Ah yes, I got that previously, then I deleted the new line. I thought it wasn't important. :D